docs/COMMAND_LINE_ARGS.md

```diff
@@ -1,32 +1,7 @@
-Command Line Arguments
-========================
-# General
+# Command Line Arguments
```

Comment:
For consistency: 

```suggestion
# Command line arguments
```

Reply:
done in bf938ae761742a4a7ecdf697cbbda17ebc059e9c

Reply:
done in bf938ae761742a4a7ecdf697cbbda17ebc059e9c